setup.py

```diff
@@ -8,32 +8,37 @@
 PACKAGE = "pypcap"
 VERSION = "1.1.5"
 
+
 def recursive_search_dirs(dirs, target_files):
     """Recursive search directories"""
     for d in dirs:
         r = recursive_search(d, target_files)
         if r:
             return r
 
+
 def recursive_search(path, target_files):
     """Recursively search for files"""
     for root, _dirs, files in os.walk(path):
         for filename in files:
             if filename in target_files:
                 return os.path.join(root, filename)
 
+
 def get_extension():
     # A list of all the possible search directories
     dirs = ['/usr', sys.prefix] + glob.glob('/opt/libpcap*') + \
         glob.glob('../libpcap*') + glob.glob('../wpdpack*') + \
-        glob.glob('/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/*')
+        glob.glob('/Applications/Xcode.app/Contents/Developer/Platforms/' +
+                  'MacOSX.platform/Developer/SDKs/*')
 
     for d in dirs:
-        # This makes sure that we first search inside of */include/pcap
-        search_dirs = [os.path.join(d, 'usr', 'include', 'pcap'),
-                    os.path.join(d, 'include', 'pcap'),
-                    os.path.join(d, 'local', 'include', 'pcap'),
-                    d]
+        search_dirs = [
+            os.path.join(d, 'local', 'include'),
+            os.path.join(d, 'usr', 'include'),
+            os.path.join(d, 'include'),
+            d
+        ]
```

Comment:
There are two pcap.h files in libpcap (hence, winpcap too) - in the "include" root and in the "include/pcap" (the former refers to the latter). It's fine in non-Windows OS to use only "include/pcap", but unfortunately for Windows pcap.h uses [pcap-stdinc.h](https://github.com/the-tcpdump-group/libpcap/blob/master/pcap/pcap.h#L75) and it's located in the "include" root. Considering the commonly used C practices (to use "include" only dirs, not subdirs), I've changed the referred code to be cross-platform (I've verified the installation in Win10, macOS 10.12 and CentOS 6.6 x86_64).